Azure-Samples/aks-store-demo
# refactor: updating azd to deploy app via helm

## Purpose

* Azure Developer CLI version 1.6.0+ now has support for deploying applications to AKS via Helm and this change will use built-in support for Helm over custom predeploy azd-hooks. 

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
* git clone https://github.com/pauldotyu/aks-store-demo.git && cd aks-store-demo
* Ensure azd version 1.6.0 or later is installed
* Enable Helm support with the command: `azd config set alpha.aks.helm on`
* Run the deployment with the command: `azd up`
* Optionally add environment variables outlined in [Azd doc](./docs/azd.md) to test additional capabilities

* Test the code
* Azd should output the URL of the store-admin site in the terminal
* Click the link to load the store-admin page

## What to Check
Verify that the following are valid
* Ensure the site functions as expected

## Other Information
